cram_pr2/cram_pr2_pick_place_demo/src/projection-demo.lisp

```diff
@@ -70,8 +76,37 @@
     (:spoon . ((-0.78 1.5 0.86) (0 0 0 1)))
     (:milk . ((-0.75 1.7 0.95) (0 0 0.7071 0.7071)))))
 
-
-(defun spawn-objects-on-sink-counter (&optional (spawning-poses *object-spawning-poses*))
+(defun make-poses-relative (&optional (spawning-poses *object-spawning-poses*))
+  (let* ((object-types '(:breakfast-cereal :cup :bowl :spoon :milk))
+         (map->surface (cl-transforms:pose->transform
```

Comment:
Please don't use this notation: `map->surface`, it can be misleading. A transformation from `map` frame to `surface` frame can be understood as, you take a coordinate in the map frame: `mapP` and multiply it with your transform `Tmap->surface` and get a point in the surface frame `surP = Tmap->sur * mapP`. But this transform does the opposite: you input a point in the surface frame and get as an output a point in map. So please use a different notation: I prefer `map-T-surface` for a transform and `map-P-surface` for a pose. The only problem with this is that our IDE doesn't like capital letters inside of variable names.